ts/components/session/ActionsPanel.tsx

```diff
@@ -230,13 +295,20 @@ export const ActionsPanel = () => {
 
   return (
     <div className="module-left-pane__sections-container">
-      <Section type={SectionType.Profile} avatarPath={ourPrimaryConversation.avatarPath} />
-      <Section type={SectionType.Message} />
-      <Section type={SectionType.Contact} />
-      <Section type={SectionType.Settings} />
+      <Section
+        setModal={setModal}
+        type={SectionType.Profile}
+        avatarPath={ourPrimaryConversation.avatarPath}
+      />
+      <Section setModal={setModal} type={SectionType.Message} />
+      <Section setModal={setModal} type={SectionType.Contact} />
+      <Section setModal={setModal} type={SectionType.Settings} />
+      {modal ? modal : null}
 
       <SessionToastContainer />
-      <Section type={SectionType.Moon} />
+
+      <Section setModal={setModal} type={SectionType.PathIndicator} hasOnionPath={hasOnionPath} />
+      <Section setModal={setModal} type={SectionType.Moon} />
```

Comment:
maybe make thr setModal props optional so we don't need to pass it when not needed?